templates/myid.j2

```diff
@@ -1 +1,9 @@
+{% if install_mode == "standalone" %}
 {{ zoo_id }}
+{% elif install_mode == "cluster" %}
+{% for server in play_hosts %}
```

Comment:
* We don't need to know the node order they will be executed, `id` is only determined by the node order in the inventory, which is fixed. No matter how the execution performs from one node to another, it does no effect on the `id`.

* If you execute a playbook by limiting the single host, you can use the default mode or standalone mode explicitly.